### PR TITLE
Handle the case where there's only one invoice in the batch

### PR DESCRIPTION
**Pull request by luislezcair**

### CODE REVIEW

File: lib/bravo/bill.rb [hunk 1]
```diff
@@ -113,6 +113,9 @@ def setup_response(response)
       request_header  = body['FeCAEReq']['FeCabReq']
       request_detail  = body['FeCAEReq']['FeDetReq']['FECAEDetRequest']
 
+      # If there's only one invoice in the batch, put it in an array
+      response_detail = [response_detail] unless response_detail.class == Array
```
Comment (horacio) [inline lib/bravo/bill.rb]: You could also use `Array()` to wrap the `response_detail` thus avoiding the conditional and the comparation. What do you think?

**Comment by horacio** [comment]
Hey Luis,

Thanks for taking the time to write this PR. I left some comments on your code. Please, let me know what you think, and let's move on to get this merged. :-)

H.

**Comment by luislezcair** [comment]
Hello.
I tried what you suggest. The problem is that when there's only one invoice `response_detail` is a just hash and wraping it in Array() gives an array of arrays in the form `[[:symbol, value], [:symbol2, value2]]`.

Later in the code the method `invoices_result` expects an Array of Hashes, so it breaks. 
I'm no Ruby expert so there might be a better way to handle that case. May be in `invoices_result`?

**Comment by horacio** [comment]
Ah, my bad. You're right. We could do, instead: `a.respond_to?(:to_ary) ? a : [a]`, blending the conditional and the comparation. But I'm fine with your approach, too. :-)

**Comment by luislezcair** [comment]
That's it. Your way looks more Ruby-like :D

**Comment by luislezcair** [comment]
I squashed both commits into one

**Comment by horacio** [comment]
Thanks, Luis! :star2:

**Pull request merged**